Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -1763,6 +1772,76 @@ - (void) testPushAndPullExpiredDocument_SG {
     AssertNil([savedDoc stringForKey: propertyKey]);
 }
 
+#pragma mark - SG (TLS with chain of certs)
+/**
+ - Use the `certs.pem` & `certs.key` in the `LiteCore/Replicator/tests/data/ssl_config.json`
+    - "SSLCert": "../../../../../Objective-C/Tests/Support/chain-certs/certs.pem",
+    - "SSLKey":  "../../../../../Objective-C/Tests/Support/chain-certs/certs.key",
+ */
+// Replicator with non-matching cert + sg with muliple certs
+- (void) testMultipleCertSGWithNonMatchingCert_SG {
+    id target = [self remoteEndpointWithName: @"itunes" secure: YES];
+    if (!target)
+        return;
+    
+    NSURL* url = [NSURL URLWithString: @"wss://localhost:4986/itunes"];
```

Comment:
I think there is no need to test with itunes which is big. Testing with scratch (empty) is fine.